gui/wxpython/datacatalog/tree.py

```diff
@@ -994,10 +1002,15 @@ def CreateLocation(self, grassdb_node):
                                              location=location,
                                              element='location',
                                              action='new')
+            self.SwitchMapset(grassdatabase, location, mapset)
+            location_created = True
+        return (location_created, is_user_in_demolocation)
 
     def OnCreateLocation(self, event):
         """Create new location"""
-        self.CreateLocation(self.selected_grassdb[0])
+        location_created, is_user_in_demolocation = (
+            self.CreateLocation(self.selected_grassdb[0])
+        )
```

Comment:
There is no need to assign the variables if you are not using them, but more importantly, this should be solved in a different way as context menu on the database node does not trigger the info bar. It also makes the two functions in tree and catalog diverge too much (two different non-trivial functions which should be the same and are not).

What do you think @petrasovaa @landam, leave for another PR or resolve now?